src/users/tests.py

```diff
@@ -214,17 +214,22 @@ class MentorsSearchTest(APITestCase):
     mentors_search_url = reverse('users:mentors_search')
     def setUp(self):
 
-        self.major = factories.MajorFactory(name='Major')
+        self.major = factories.MajorFactory(name='Test_Major0')
         self.mentor = factories.MentorFactory(major=self.major)
 
-        self.major1 = factories.MajorFactory(name='Test_Major')
-        self.profile1 = factories.ProfileFactory(year='1st')
+        self.user1 = factories.UserFactory(first_name='Unique_First', last_name='Unique_Last')
+        self.major1 = factories.MajorFactory(name='Test_Major1')
+        self.profile1 = factories.ProfileFactory(year='1st', user=self.user1)
         self.mentor1 = factories.MentorFactory(major=self.major1, profile=self.profile1)
 
         self.major2 = factories.MajorFactory(name='Test_Major2')
         self.profile2 = factories.ProfileFactory(year='2nd')
         self.mentor2 = factories.MentorFactory(major=self.major2, profile=self.profile2)
 
+        self.major3 = factories.MajorFactory(name='Test_Major3')
+        self.profile3 = factories.ProfileFactory(year='2nd')
```

Comment:
I know this was already merged, but these really should be constants instead of '2nd'.